packages/compat/package.json

```diff
@@ -24,59 +24,59 @@
     "test": "jest"
   },
   "dependencies": {
-    "@babel/code-frame": "^7.10.4",
-    "@babel/core": "^7.12.3",
+    "@babel/code-frame": "^7.14.5",
+    "@babel/core": "^7.14.5",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/traverse": "^7.12.1",
-    "@babel/types": "^7.12.1",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/traverse": "^7.14.5",
+    "@babel/types": "^7.14.5",
     "@embroider/macros": "0.41.0",
     "@types/babel__code-frame": "^7.0.2",
-    "@types/yargs": "^15.0.9",
+    "@types/yargs": "^17.0.0",
     "assert-never": "^1.1.0",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "babylon": "^6.18.0",
     "bind-decorator": "^1.0.11",
-    "broccoli": "^3.4.2",
-    "broccoli-concat": "^3.7.3",
+    "broccoli": "^3.5.2",
+    "broccoli-concat": "^4.2.5",
     "broccoli-file-creator": "^2.1.1",
-    "broccoli-funnel": "^3.0.5",
-    "broccoli-merge-trees": "^3.0.0",
+    "broccoli-funnel": "^3.0.7",
+    "broccoli-merge-trees": "^4.2.0",
     "broccoli-persistent-filter": "^3.1.2",
     "broccoli-plugin": "^4.0.7",
-    "broccoli-source": "^3.0.0",
-    "chalk": "^4.1.0",
-    "debug": "^3.1.0",
-    "fs-extra": "^7.0.0",
-    "fs-tree-diff": "^2.0.0",
-    "jsdom": "^16.4.0",
-    "lodash": "^4.17.10",
+    "broccoli-source": "^3.0.1",
+    "chalk": "^4.1.1",
+    "debug": "^4.3.2",
+    "fs-extra": "^10.0.0",
```

Comment:
I don't think we can bump to `fs-extra@10` yet (until we drop support for Node 10). I think we can absolutely drop support, but it ought to be a dedicated PR.

Reply:
when should we do that?